src/styles/reset.css

```diff
@@ -0,0 +1,101 @@
+/* Reset and base styles  */
```

Comment:
did you download it? 
I don't think you had to, there're not so many styles to reset, and this resetter could be damaging, however, up to you

Reply:
yep, i downloaded this, but maybe in further projects i will use another one or my own, because i would like to have an reset css file to apply it on projects i want)